source/guides/dashboard/slack-integration.md

```diff
@@ -0,0 +1,112 @@
+1. Under **Actions**, select your **Delete** for each Slack channel you want deleted.
+
+# Per-project configuration
+
+If your organization has multiple teams working on separate projects, you can tailor the Slack notifications of each project to match your teams' needs.
+
+## Add a new Slack channel
+
+You can have the Cypress Dashboard post run results for a specific project to an additional channel.
+
+1. Select your organization in the organization switcher.
+  {% imgTag /img/dashboard/select-cypress-organization.png "Select an organization" width-600 %}
+1. Select the project you wish to integrate with a GitHub repository.
+  {% imgTag /img/dashboard/select-cypress-project.png "Select a project" %}
+1. Go to the project's settings page.
+  {% imgTag /img/dashboard/visit-project-settings.png "Visit project settings" %}
+1. Scroll down to the **Slack Integration** section.
+1. Click **Add Slack Channel**.
+1. You'll see a popup window that allows you to choose the channel to associate with the project.
+
+## Set notification preferences
+
+By default, the Cypress Dashboard will post a Slack message to each configured channel only for failing runs. You cannot override the notification preferences for the global organization channels. If you'd like to change these preferences:
+
+1. Navigate to the **Integrations** page for the project with the installed Slack integration.
+1. Scroll down to the **Slack Integration** section.
+1. Under **Notifications**, select your preference for each Slack channel:
+  - **All runs**: will notify on all runs (including passed)
+  - **Failed runs only**: will only notify on runs with a fail status.
+
+## Mute a channel
+
+If you want the Cypress Dashboard to temporarily stop posting Slack messages to a certain channel, you can **Mute** that channel. This allows you to easily pause and resume notifications for a specific channel without losing the configuration you’ve put in place. You can even mute the messages for the global organization channels!
+
+1. Navigate to the **Integrations** page for the project with the installed Slack integration.
+1. Scroll down to the **Slack Integration** section.
+1. Under **Actions**, select your **Mute** for each Slack channel you want muted.
+
+## Remove a channel
+
+You can have the Cypress Dashboard stop posting notifications to a channel. You cannot delete the global notification channels from a project.
+
+1. Navigate to the **Integrations** page for the project with the installed Slack integration.
+1. Scroll down to the **Slack Integration** section.
+1. Under **Actions**, select your **Delete** for each Slack channel you want deleted.
+
+# Remove the integration
+
+You can completely remove the Slack integration from your workspace. This will remove the @cypress bot from your workspace and will delete all of the Slack configurations you’ve set in the Cypress Dashboard. You cannot undo this, but you will be able to install the Slack integration again in the future.
```

Comment:
```suggestion
You can completely remove the Slack Integration from your workspace. This will remove the @cypress bot from your workspace and will delete all of the Slack configurations you’ve set in the Cypress Dashboard. You cannot undo this, but you will be able to install the Slack Integration again in the future.
```